Python Examples/TestMonitor/test_data_manager_client.py

```diff
@@ -0,0 +1,140 @@
+import sys
+import requests
+
+create_results_route = "nitestmonitor/v2/results"
+create_steps_route = "nitestmonitor/v2/steps"
+update_results_route = "nitestmonitor/v2/update-results"
+update_steps_route = "nitestmonitor/v2/update-steps"
+
+def print_usage_and_exit(error:str):
```

Comment:
I'd prefer you put the usage information in the main and construct. 

If we want a better command line interface we can use the [click](https://pypi.org/project/click/) pypi package. That would also make it easy to use environment variables for the CLI options, which is handy for the API key. We can also decide this is just an example and doesn't need to be very pretty.

Reply:
Updated the script to use click pypi package.